javascript-carousel/main.js

```diff
@@ -0,0 +1,255 @@
+    $jigglypuff.className = "jigglypuff"
+    $circleOne.className = "far fa-circle circle-btn circle-btn-one";
+    $circleTwo.className = "far fa-circle circle-btn circle-btn-two";
+    $circleThree.className = "far fa-circle circle-btn circle-btn-three";
+    $circleFour.className = "far fa-circle circle-btn circle-btn-four";
+    $circleFive.className = "fas fa-circle circle-btn circle-btn-five";
+  } else if ($charmander.className === "charmander") {
+    $bulbasaur.className = "bulbasaur";
+    $charmander.className = "charmander hidden";
+    $squirtle.className = "squirtle hidden";
+    $pikachu.className = "pikachu hidden";
+    $jigglypuff.className = "jigglypuff hidden"
+    $circleOne.className = "fas fa-circle circle-btn circle-btn-one";
+    $circleTwo.className = "far fa-circle circle-btn circle-btn-two";
+    $circleThree.className = "far fa-circle circle-btn circle-btn-three";
+    $circleFour.className = "far fa-circle circle-btn circle-btn-four";
+    $circleFive.className = "far fa-circle circle-btn circle-btn-five";
+  } else if ($squirtle.className === "squirtle") {
+    $bulbasaur.className = "bulbasaur hidden";
+    $charmander.className = "charmander";
+    $squirtle.className = "squirtle hidden";
+    $pikachu.className = "pikachu hidden";
+    $jigglypuff.className = "jigglypuff hidden"
+    $circleOne.className = "far fa-circle circle-btn circle-btn-one";
+    $circleTwo.className = "fas fa-circle circle-btn circle-btn-two";
+    $circleThree.className = "far fa-circle circle-btn circle-btn-three";
+    $circleFour.className = "far fa-circle circle-btn circle-btn-four";
+    $circleFive.className = "far fa-circle circle-btn circle-btn-five";
+  } else if ($pikachu.className === "pikachu") {
+    $bulbasaur.className = "bulbasaur hidden";
+    $charmander.className = "charmander hidden";
+    $squirtle.className = "squirtle";
+    $pikachu.className = "pikachu hidden";
+    $jigglypuff.className = "jigglypuff hidden"
+    $circleOne.className = "far fa-circle circle-btn circle-btn-one";
+    $circleTwo.className = "far fa-circle circle-btn circle-btn-two";
+    $circleThree.className = "fas fa-circle circle-btn circle-btn-three";
+    $circleFour.className = "far fa-circle circle-btn circle-btn-four";
+    $circleFive.className = "far fa-circle circle-btn circle-btn-five";
+  } else if ($jigglypuff.className === "jigglypuff") {
+    $bulbasaur.className = "bulbasaur hidden";
+    $charmander.className = "charmander hidden";
+    $squirtle.className = "squirtle hidden";
+    $pikachu.className = "pikachu";
+    $jigglypuff.className = "jigglypuff hidden"
+    $circleOne.className = "far fa-circle circle-btn circle-btn-one";
+    $circleTwo.className = "far fa-circle circle-btn circle-btn-two";
+    $circleThree.className = "far fa-circle circle-btn circle-btn-three";
+    $circleFour.className = "fas fa-circle circle-btn circle-btn-four";
+    $circleFive.className = "far fa-circle circle-btn circle-btn-five";
+  }
+})
```

Comment:
This logic can be minified using a loop and some sort of internal logic in your code.  Keep track of the position of the images in a global variable so you don't ask the DOM what is currently happening.